lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -505,6 +562,69 @@ static const std::string Bf16_to_Fp8E5M2(bool hasNativeFP) {
   return ret;
 }
 #endif
+
+// ROCM type conversion between fp8 and bf16
+#ifdef USE_ROCM
+
+static Value cvtBf16ToFp32(Location loc, 
```

Comment:
There is already a function for converting bf16 to fp32 in this file: convertBf16ToFp32

Reply:
Thanks, fixed.